integration_tests/src/main/python/delta_lake_delete_test.py

```diff
@@ -157,7 +160,10 @@ def generate_dest_data(spark):
 @pytest.mark.parametrize("partition_columns", [None, ["a"]], ids=idfn)
 @pytest.mark.skipif(is_before_spark_320(), reason="Delta Lake writes are not supported before Spark 3.2.x")
 @datagen_overrides(seed=0, permanent=True, reason='https://github.com/NVIDIA/spark-rapids/issues/9884')
-def test_delta_delete_rows(spark_tmp_path, use_cdf, partition_columns):
+@pytest.mark.parametrize("enable_deletion_vectors", deletion_vector_values_with_reasons(
+                                        enabled_xfail_reason="https://github.com/NVIDIA/spark-rapids/issues/12041",
+                                        disabled_xfail_reason="https://github.com/NVIDIA/spark-rapids/issues/12047"), ids=idfn)
```

Comment:
But with it disabled only fails on 14.3. This code has this marked as xfail for all versions of delta lake.

Reply:
Fixed